docs/learn/chrysalis/core-concepts/switch-to-UTXO.md

```diff
@@ -1,2 +1,76 @@
-extract article fom https://govern.iota.org/t/switching-to-utxo-model-for-balances-colored-coins-easier-conflict-resolution/229
-or TIP
+---
+description: Learn about the switch from account balance models to Unspent Transaction Output (UTXO) in IOTA.
+keywords:
+  [
+    'IOTA',
+    'UTXO',
+    'Balance Models',
+    'Conflicts',
+    'Reattachments',
+    'Colored Coins',
+  ]
+---
+
+# Switch to UTXO
+
+UTXO means "unspent transaction output".
+This means that instead of tracking the balances of addresses, you also keep track of where balances came from and
+where they go when they are spent.
+
+## The Problem with the Balance Mode
+
+Before Chrysalis, IOTA used an account balance model to track addresses.
+This meant every address had a single value associated with it, its current balance.
+This meant the ledger tracked **Address1**, which was associated with **Balance1**, **Address2** was associated with \*
+\*Balance2\*\*, and so on.
+
+### Double-spends
+
+The main problem with this model was dealing with double-spends, also known as conflicts.
+It took a lot of work to determine which of multiple transactions performed a double spend and which used legitimate
+funds.
+
+### Reatachments
+
+Another problem with using an account balance model is reattachments.
+If somebody ever receives funds at an address already used to send tokens, anybody can just reattach the previous
+transaction and empty the address again,
+even if they do not control the private key.
+
+## The UTXO Solution
+
+Since Chrysalis, IOTA switched to the UTXO model to keep track of balances.
+Every address contains its total balance and multiple sub-balances with a marker indicating precisely which transaction
+created the funds.
+Every coin on an address can be uniquely identifiable, and every transaction names the exact coins it wants to move.
```

Comment:
```suggestion
```

I would remove this. It is confusing at best and coins if kept should be replaces with Native tokens. But still weird